File: HW01/CPP/src/TableTennisTable.cpp
#include "../include/TableTennisTable.h"
#include <iostream>
#include <string>
using namespace std;

TableTennisTable::TableTennisTable(string material, double price) : IndoorItem(price){
    this->material = material;
    this->price = price;
}

void TableTennisTable::display() {
    cout << "$" << this->price << " - This table tennis table is made of " << this->material;
}

File: HW01/PYTHON/project/TableTennisTable.py
from .IndoorItem import IndoorItem

class TableTennisTable(IndoorItem):
    def __init__(self, price, material):
        super().__init__(price)
        self.material = material
    
    def display(self):
        print(f"${self.price} - This table tennis paddle is made of {self.material}", end="")